catroid/src/org/catrobat/catroid/formulaeditor/Formula.java

```diff
@@ -272,8 +272,6 @@ public String getResultForComputeDialog(Context context) {
 			} catch (InterpretationException interpretationException) {
 				return "ERROR";
 			}
-			interpretationResult *= 100;
-			interpretationResult = (double) (Math.round(interpretationResult) / 100f);
```

Comment:
what do you mean with current output: rounded up to two decimals?
Currently: https://www.dropbox.com/s/ko2sskxlsv5zd6a/Screenshot_2016-04-27-17-18-41.png?dl=0 
all decimal places but the wrong value - we only changed the value to the right one: 3.141592653589793 - why do you need input from the ux?